docs/css/example-search-styles.css

```diff
@@ -3722,7 +3727,7 @@ dl dd.multi-item-list .nypl-basic-table,
 
 .nypl-results-summary {
   display: inline-block;
-  max-width: 65%; }
+  flex-grow: 2; }
```

Comment:
Fancy 🎩

Reply:
maybe I should use `@supports` and keep it in?

Reply:
It'd be good to test the PR in multiple browsers and then see if we do need to use `@supports`. But, I'm liking it and don't think it'd hurt to check for it. You'd then need another set of rules for when it isn't supported, no?

Reply:
yeah. we'd basically have to have the previous work as the fall back.